modules/distr-tracing-tempo-config-multitenancy.adoc

```diff
@@ -159,9 +159,21 @@ spec:
             authenticator: bearertokenauth
           headers:
             X-Scope-OrgID: "dev"
+        otlphttp/dev: # <2>
+          endpoint: tempo-simplest-gateway.tempo.svc.cluster.local:8080
```

Comment:
Endpoint should be as below:
endpoint: https://tempo-simplest-gateway.chainsaw-multitenancy.svc.cluster.local:8080/api/traces/v1/dev

Reply:
I'm glad I asked for your review, Ishwar.
I've replaced the endpoint value with the one you shared above.
Thank you.